web/web.go

```diff
@@ -145,14 +145,8 @@ func root(c *api.Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(c.Session.UserId) == 0 {
```

Comment:
Why remove this functionality? A logged in user going to root probably doesn't want to see the signup page and will be confused.